### PR TITLE
Consolidate 1.8 and 1.9 Dockerfiles

### PR DESCRIPTION
**Pull request by jwendell**
Do not build a `2.1-1.9` Dockerfile. Build only 2.1, based on the 1.9 Dockerfile.

### CODE REVIEW

**Comment by jwendell** [comment]
Let's keep `2.1-1.9` jobs for now. They will not be triggered anyway. We can clean them up later, in a follow up.

File: prow/config.gen.yaml [hunk 1]
```diff
@@ -1561,8 +1561,9 @@ presubmits:
           value: /tmp/cache
         - name: GOCACHE
           value: /tmp/cache
-        - name: ISTIO_ENVOY_BASE_URL
-          value: https://storage.googleapis.com/maistra-prow-testing/proxy
+        # FIXME: disabled until we have a proxy build
+        #  - name: ISTIO_ENVOY_BASE_URL
+        #  value: https://storage.googleapis.com/maistra-prow-testing/proxy
```
Comment (brian-avery) [inline prow/config.gen.yaml:1566]: Don't we have a proxy build?

Comment (jwendell) [reply prow/config.gen.yaml:1566]: We don't have them building on github yet, therefore there are no 1.9 binaries on google storage yet.
I'm talking to Dmitri about this. Meanwhile, we should keep using upstream proxy.

Comment (brian-avery) [reply prow/config.gen.yaml:1566]: Ah. Nevermind. I was thinking dailies.

**Review by brian-avery — APPROVED** [review]

**Pull request merged**